Vagrantfile

```diff
@@ -0,0 +1,101 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+Vagrant.configure("2") do |config|
+  # The most common configuration options are documented and commented below.
+  # For a complete reference, please see the online documentation at
+  # https://docs.vagrantup.com.
+
+  # Every Vagrant development environment requires a box. You can search for
+  # boxes at https://vagrantcloud.com/search.
```

Comment:
You should delete all these comments now that you've got the Vagrantfile working. They're just obscuring your actual configuration.

At most you should keep
```
  # Please see the online documentation at
  # https://docs.vagrantup.com.
```

Reply:
Done